test/e2e-api/admin/__snapshots__/members.test.js.snap

```diff
@@ -2696,7 +2696,7 @@ exports[`Members API Can subscribe to a newsletter 5: [headers] 1`] = `
 Object {
   "access-control-allow-origin": "http://127.0.0.1:2369",
   "cache-control": "no-cache, private, no-store, must-revalidate, max-stale=0, post-check=0, pre-check=0",
-  "content-length": "4398",
+  "content-length": "4326",
```

Comment:
tl;dr; should be okay, because we drop the subscribed property in the member activity feed.

Reply:
Yeah, this is because `subscribed` leaks through the activity feed API endpoint as I mentioned in Slack: https://ghost.slack.com/archives/C02G9E68C/p1651762433487199

Reply:
Merging but LMK if you think this is a concern 🙂